examples/microViT.py

```diff
@@ -248,7 +248,7 @@ def test_step(self, batch, _):
 
     # compute total number of steps
     batch_size = BATCH * GPUS
-    steps = dm.num_samples // batch_size * MAX_EPOCHS
```

Comment:
this should have been fixed earlier, I had a PR stack and somehow lost it with rebases..